ios/FluentUI/Badge Field/BadgeView.swift

```diff
@@ -11,11 +11,17 @@ open class BadgeViewDataSource: NSObject {
     @objc open var text: String
     @objc open var style: BadgeView.Style
     @objc open var size: BadgeView.Size
+    @objc open var customView: UIView?
+    open var customViewVerticalPadding: CGFloat?
 
-    @objc public init(text: String, style: BadgeView.Style = .default, size: BadgeView.Size = .medium) {
+    @objc public init(text: String, style: BadgeView.Style = .default, size: BadgeView.Size = .medium, customView: UIView? = nil, customViewVerticalPadding: NSNumber? = nil) {
```

Comment:
1. we should be using convenience initilization so that we don't disrupt existing objc clients who is using badgeview. 
2. we should be using edgeinset type of datastructure to pass down any padding information.

Reply:
Add a convenience initilization and use 3 `NSNumber?` instead of `UIEdgeInsets` for these reason:
1. Objc doesn't support optianal `UIEdgeInsets`
2. If users only want to custom vertival padding, they don't know how to set the left and right value of insets
3. We need to enforce `paddingTop` equals to `paddingBottom`. It's hard to layout custom view if `customView.height` + `customView.paddingTop` + `customView.paddingBottom` < `badgeView.height` and `paddingTop` isn't equal to `paddingBottom`. If user have special design, they can use a wrapper view to walk around.

Reply:
I haven't finished reviewing the rest of the PR, but the first issue pointed out by @harrieshin is still applicable here. We can't change initializers that are used by Objective-C, since they can't recognize default values. Please add a `convenience` initializer with the additional parameters, and leave the original as-is.

Reply:
@mischreiber It's added in the latest iteration

Reply:
Updated